cmd/init.go

```diff
@@ -38,6 +38,11 @@ func createInitCommand() *cobra.Command {
     address: "127.0.0.1"
     grpc_port: 19090
     http_port: 19091
+  monitoring_info:
+    promtheus:
```

Comment:
there's typo `promtheus`
It should be `prometheus`

Reply:
Fixed typo.

> Commit: d1ad4abe13f97e8132d823831909d388ef4862a2